code/modules/clothing/head/armor.dm

```diff
@@ -545,7 +545,7 @@
 	desc = "May God guide you."
 	icon_state = "crusader_hemet"
 	item_state = "crusader_hemet"
-	flags_inv = HIDEEARS|HIDEEYES|BLOCKHAIR
+	flags_inv = HIDEFACE|BLOCKHAIR
```

Comment:
```suggestion
	body_parts_covered = HEAD|FACE|EYES|EARS
	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|BLOCKHAIR
```
While comparing this to similar helmets, I noticed it was missing a lot of flags.